.github/workflows/deploy.yml

```diff
@@ -366,7 +366,7 @@ jobs:
       fail-fast: true
       matrix:
         # run copies of the current job in parallel
-        containers: [1, 2, 3, 4, 5, 6]
+        containers: [1, 2, 3, 4, 5, 6, 7, 8 ]
```

Comment:
a bit farther down in the same file, we collect the cypress coverage reports.  there are lines like
```
cp ./cypress-json-coverage-6/coverage-final.json ./cypress_temp/from-cypress-6.json
```
I think you'll need to add a couple more there, for the new containers.